lib/model/message.dart

```diff
@@ -35,6 +35,37 @@ mixin MessageStore {
   /// All [Message] objects in the resulting list will be present in
   /// [this.messages].
   void reconcileMessages(List<Message> messages);
+
+  /// Whether the current edit request, if any, has failed.
```

Comment:
This reads to me as if there's one "current edit request" on the store. How about:
```suggestion
  /// Whether the current edit request for the given message, if any, has failed.
```